docs/site/Sequence.md

```diff
@@ -217,30 +217,106 @@ namespace.
 
 ### Query string parameters
 
-{% include content/tbd.html %}
+OAI 3.0.x describes the data from a request’s header, query and path in an
+operation specification’s parameters property. In a Controller method, such an
+argument is typically decorated by @param(). We've made multiple shortcuts
+available to the `@param()` decorator in the form of
+`@param.<http_source>.<OAI_primitive_type>`. Using this notation, query string
+parameters can be described as `@param.query.<OAI_primitive_type>`. Here is an
+example of a controller method which retrieves a Note model instance by
+obtaining the `id` from the query string object.
 
-How to get query string param values.
+```ts
+@get('/notes/{id}', {
+  responses: {
+    '200': {
+      description: 'Note model instance',
+      content: {'application/json': {schema: {'x-ts-type': Note}}},
+    },
+  },
+})
+async findById(@param.query.string('id') id: string): Promise<Note> {
+  return await this.noteRepository.findById(id);
+}
+```
+
+You can also specify a parameter which is an object value encoded as a JSON
+string or in multiple nested keys. For a JSON string, a sample value would be
+`location={"lang": 23.414, "lat": -98.1515}`. For the same `location` object, it
```

Comment:
A question: is it meant to be the string value get from the HTTP client?

If we are showing an example value from in the API explorer input field, probably wrap key&value with double quotes.

Reply:
Actually I took this straight from API explorer and I think they're not wrapped because we define lat/long as number values.